test/surface_test.py

```diff
@@ -284,8 +309,27 @@ def test_subsurface_rle2(self):
         s2 = s1.subsurface(sub_rect)
         s2.fill(color2)
         s0.blit(s2, (0, 0))
-        self.assertTrue(s1.get_flags() & pygame.RLEACCEL)
-        self.assertTrue(not s2.get_flags() & pygame.RLEACCEL)
+        self.assertTrue(s1.get_flags() & pygame.RLEACCELOK)
+        self.assertTrue(not s2.get_flags() & pygame.RLEACCELOK)
+
+    def test_solarwolf_rle_usage(self):
+
+        def optimize(img):
+            clear = img.get_colorkey()
+            img.set_colorkey(clear, RLEACCEL)
+            return img.convert()
+
+        pygame.display.init()
+        try:
+            window_surf = pygame.display.set_mode((640, 480))
+
+            image = pygame.image.load(example_path(os.path.join("data",
+                                                        "ship-up-boost1.png")))
```

Comment:
Instead of adding this file, we could use alien1.png?

```
alien1.png: PNG image data, 80 x 71, 8-bit colormap, non-interlaced
ship-up-boost1.png: PNG image data, 128 x 32, 8-bit colormap, non-interlaced
```

Reply:
Sure, I was just overzealously copying from solarwolf to begin with to reproduce the crash.